### PR TITLE
Assert library string comparison

### PR DESCRIPTION
**Pull request by 0mkara**
- Compare strings using `keccak256`

### CODE REVIEW

File: sol/tests.sol.js [hunk 1]
```diff
@@ -50,11 +50,10 @@ library Assert {
...
   }
 
-  // TODO: needs to be convert to bytes first to be comparable
-  //function equal(string a, string b, string message) public returns (bool result) {
-  //  result = (a == b);
-  //  emit AssertionEvent(result, message);
-  //}
+  function equal(string a, string b, string message) public returns (bool result) {
+     result = (keccak256(a) == keccak256(b));
+     AssertionEvent(result, message);
```
Comment (matt9ucci) [inline sol/tests.sol.js]: `emit AssertionEvent(result, message);`
see #18

File: sol/tests.sol.js [hunk 2]
```diff
@@ -93,11 +92,10 @@ library Assert {
...
   }
 
-  // TODO: needs to be convert to bytes first to be comparable
-  //function notEqual(string a, string b, string message) public returns (bool result) {
-  //  result = (a != b);
-  //  emit AssertionEvent(result, message);
-  //}
+  function notEqual(string a, string b, string message) public returns (bool result) {
+    result = (keccak256(a) != keccak256(b));
+    AssertionEvent(result, message);
```
Comment (matt9ucci) [inline sol/tests.sol.js]: `emit AssertionEvent(result, message);`
see #18

**Review by matt9ucci — CHANGES_REQUESTED** [review]
You don't have to remove "emit".

**Comment by 0mkara** [comment]
@matt9ucci yes I will add `emit` to all of them. This repo is moving into remix https://github.com/ethereum/remix/pull/924. Once that PR is merged I will update my PR with `emit`. Thanks for pointing them out.

**Pull request merged**